doc/tutorials/bare_flame_game.md

```diff
@@ -96,13 +101,13 @@ version: 0.0.0
 publish_to: none
 
 environment:
-  sdk: ^2.16.0
-  flutter: ^2.10.0
+  sdk: ^3.0.0
+  flutter: ^3.13.0
 
 dependencies:
   flutter:
     sdk: flutter
-  flame: ^--VERSION--
+  flame: ^1.9.1
```

Comment:
This should be kept as it was, so that we don't have to update it all the time

Reply:
reverted